src/libraries/System.Formats.Tar/tests/Manual/ManualTests.cs

```diff
@@ -20,10 +20,8 @@ public static IEnumerable<object[]> WriteEntry_LongFileSize_TheoryData()
             foreach (TarEntryFormat entryFormat in new[] { TarEntryFormat.V7, TarEntryFormat.Ustar, TarEntryFormat.Gnu, TarEntryFormat.Pax })
             {
                 yield return new object[] { entryFormat, LegacyMaxFileSize, unseekableStream };
+                yield return new object[] { entryFormat, LegacyMaxFileSize + 1, unseekableStream };
             }
-
-            // Pax supports unlimited size files.
-            yield return new object[] { TarEntryFormat.Pax, LegacyMaxFileSize + 1, unseekableStream };
```

Comment:
I'm not sure I don't understand why this was moved to the forloop above, can you please explain?

Reply:
Before the change, only the Pax format allowed the larger files. This test was updated to run against all formats.

Based on the other feedback, we'll allow the larger file range for Pax and Gnu, and I'll update this accordingly.